src/service/ampdoc-impl.js

```diff
@@ -300,6 +303,14 @@ export class AmpDoc {
   contains(node) {
     return this.getRootNode().contains(node);
   }
+
+  /**
+   * Binding of 3p analytics vendors' responses, used amp-ad-exit
+   * @return {!Object<string, !Object<string, Object<string,string>>>}
+   */
+  getIframeTransportResponses() {
```

Comment:
@lannka @keithwrightbos I investigated this further and confirmed my suspicion. When adding a response to the map, it is adding it in the context of amp-analytics-0.1.max.js, but when checking if the map contains a response it is in the context of amp-ad-exit-0.1.max.js. Since these are different source files, it is treating them as distinct classes, even though they have the same class name. Thus the responses which amp-analytics puts into the map are not visible by amp-ad-exit, *if* the map lives in either extension. So, I think it will have to live in AmpDoc (or possibly something else that is returned by a service? I can investigate that if desired.)
At response time (note FunctionLocation):
![image](https://user-images.githubusercontent.com/2455047/29679002-8a5787a6-88ce-11e7-9019-b24ac509799f.png)

At click time:
![image](https://user-images.githubusercontent.com/2455047/29679019-95ab8e72-88ce-11e7-91ac-b9f307a39a9a.png)